requirements.txt

```diff
@@ -17,7 +17,6 @@ matplotlib==3.7.0
 more-itertools==9.1.0
 mypy-extensions==1.0.0
 numpy==1.24.2
-openai-whisper @ git+https://github.com/openai/whisper.git@7858aa9c08d98f75575035ecd6481f462d66ca27
```

Comment:
I'm curious why this was removed. Could have been a mistake on my part, but just want to make sure.

Reply:
I had an issue where installing `whisper-timestamped` didn't like the specific static version of `openai-whisper`. Removed it and it seemed to infer what it needed and installed perfectly after removing. 

I could very well be in the wrong here. Try the build and let me know if it's needed.

Reply:
I think this got fixed in a separate PR. Maybe pull in the latest and fix merge conflicts?

Reply:
perfect, I can manage that.